### ISSUE TITLE
force agent data request

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6411]
ffdss just force-created agent [R. L. Johnson](https://arctos.database.museum/agents.cfm?agent_id=21348806).

@DerekSikes is there any additional info for this? 

@ArctosDB/agents-committee can we talk about the bar, I don't think it's high enough? There must be thousands of 2-initials and a common last name alive in any given year.

**Comment by DerekSikes** [comment]
Can we use verbatim agents as determiners? If not then we need to use the data we have and all I have is R. L. Johnson. To use 'unknown' instead is wrong.

**Comment by dustymc** [comment]
>verbatim agents as determiners?

With next release, sure. We'll need an identification_attributes code table request (thought there was one but I can't find it, @Jegelewicz ??)

**Comment by lin-fred** [comment]
Is this the request you are looking for https://github.com/ArctosDB/arctos/issues/5283

**Comment by dustymc** [comment]
Ah thanks @lin-fred - yes, probably, but a request is still needed. The agents discussed in there might be moved once https://github.com/ArctosDB/arctos/issues/4829 is in place.

**Comment by Jegelewicz** [comment]
Identification attributes - https://github.com/ArctosDB/arctos/issues/6243

**Comment by dustymc** [comment]
That's not (quite) it either - that's an inevitable part of https://github.com/ArctosDB/arctos/issues/5331 (which could use some testing as requested), this need to be a request for whatevre this concept should be called.

**Comment by Jegelewicz** [comment]
Yes - but I was waiting for the code table to exist....

**Comment by dustymc** [comment]
https://arctos-test.tacc.utexas.edu/info/ctDocumentation.cfm?table=ctidentification_attribute_type

It will be coming to prod very soon....

**Comment by DerekSikes** [comment]
y'all probably already know this but given the tiny data labels for insects abbreviated agent names are the norm.

Glad to see we'll have verbatim agents as an alternative (and I recall the very idea of verbatim agents originated with me back in <2012 but was never made that useful until recently).

**Comment by Jegelewicz** [comment]
@DerekSikes I thought that was why we have the [labels](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctagent_name_type#labels) name type for agents?

**Comment by dustymc** [comment]
>abbreviated agent names are the norm.

That's why https://arctos.database.museum/info/ctDocumentation.cfm?table=ctagent_name_type#labels exists, there's no need to be cryptic outside of labels, ping me if you need report help (or check MVZ's reports, that's where this originated).

>very idea of verbatim agents originated with me back in <2012

Yes, but that structure was problematic. It's awesome now because it's segregated, off by itself where it can happily be a different kind of data, which it clearly is. I don't even know who came up with the idea of attributes, but I'm certain that we'd not be where we are now without a bunch of input from a bunch of people. So thanks to you and the rest of our little Community!

**Comment by DerekSikes** [comment]
I thought that was why we have the labels name type for agents?

You misunderstand - the source data are on tiny labels already for retroactive data capture. The shortened abbreviated agent names on tiny labels are ALL we have to go on most of the time. Trust me, I would never throw away data if I have it.
